src/applications/facility-locator/api/LocatorApi.js

```diff
@@ -37,7 +37,14 @@ class LocatorApi {
         .then(
           data => resolve(data),
           error => reject(error)
-        );
+        ).catch(() => {
+          resolve({
+            errors: [{
+              code: '408',
+              }
+            ]
+          });
+        })
```

Comment:
It seemed like when `fetch` was failing with a `TypeError`, it was due to a timeout, but I'm not sure.

Reply:
My understanding is that `fetch` only fails here if the user is offline. I think it always succeeds if it gets a valid http code and you have to inspect that code if you want to return a rejected promise. Haven't verified this personally, though.

Reply:
Ah, I had the app opened in multiple tabs trying to learn more about the API errors because it takes a minute to load results sometimes. It sounds like that could've happened.